.ci/scripts/describe_system.sh

```diff
@@ -54,6 +54,10 @@ if [ -x "$(command -v icc)" ]; then
     echo "ICC:"
     icc --version
 fi
+if [ -x "$(command -v icx)" ]; then
+    echo "ICX:"
+    icx --version
+fi
```

Comment:
AR: same should be applied for sklearnex repo script

Reply:
https://github.com/intel/scikit-learn-intelex/pull/2019